### DISCUSSION TITLE
Feature 5g unlock any carrier

### DISCUSSION THREAD

**Discussion by ivanvart** [kyujin-cho/pixel-volte-patch#82] [Ideas]
I'm wondering if it's possible adding the feature to unlock 5g for any carrier, so far I read thos blocking is just software like the volte.

**Comment by kyujin-cho** [comment]
Very unlikely. Check https://github.com/kyujin-cho/pixel-volte-patch/issues/2 to track 5G issues (english posts at the end of the issue).
